packages/clarity-js/types/data.d.ts

```diff
@@ -126,7 +130,9 @@ export const enum Dimension {
     Platform = 22,
     PlatformVersion = 23,
     Brand = 24,
-    Model = 25
+    Model = 25,
+    DevicePixelRatio = 26,
```

Comment:
This is a number but floating point (and don't want to convert to int). Is that why it's a dimension?

Reply:
Yes, right.